web/client/configs/newgeostory.json

```diff
@@ -7,6 +7,11 @@
         "color": "#333333",
         "backgroundColor": "#ffffff",
         "borderColor": "#e6e6e6"
+      },
+      "overlay": {
```

Comment:
what happens if these values are not present? are these default present somewhere else?

opening an existing story has this behaviour
![image](https://user-images.githubusercontent.com/11991428/88907980-957ad980-d259-11ea-9e77-0d0b4387b647.png)

imo these defaults should not be set (only) here, @allyoucanmap do you agree?

Reply:
> opening an existing story has this behaviour

I don't see the issue in this behaviour because that represent correctly the configuration of the overlay theme. In previous version the default style toke the colors for default style from the general theme configuration so in existing story I would expect those color picker transpare or inactive but maybe I misunderstood your comment?